### ISSUE TITLE
Initial cursor size is smaller than after entering the first character

### ISSUE THREAD

**Issue by JWo1F** [kyle-n/HighlightedTextView#73]
**Describe the bug**
If the initial text is set to an empty string, the cursor is smaller than after the first character is entered. If all characters are erased, the cursor size is displayed correctly.

**To Reproduce**
1. Create `@State private var message = ""`
2. Use as `HighlightedTextEditor(text: $message, highlightRules: rules).frame(minHeight: 50)`
3. Now you can see small cursor
4. Enter one character and you'll see normal cursor
5. Remove the character and you'll see normal cursos

**Expected behavior**
The cursor is always displayed at the same size

**Screenshots**

https://github.com/kyle-n/HighlightedTextEditor/assets/1540715/56bc7aef-c040-4fdd-afc4-0b6217374d67

**Environment**
Please include:
- Exact version of iOS / iPadOS / macOS where your app was running when you saw the bug
  - iOS 16.4, iPhone 11 Pro
- Is your app using AppKit or UIKit? If your app is for iOS or Mac Catalyst, it is UIKit. If it is for macOS natively, it is using AppKit.
  - UIKit

**Additional context**
I use flutter, but all the work happens in an independent UIViewController

**Comment by kyle-n** [comment]
Confirmed with latest version of HLTE. Issue also occurs in the AppKit editor.

**Comment by kyle-n** [comment]
Fixed in v2.1.2. Use the `.introspect()` callback:

```swift
let baseFont = UIFont.systemFont(ofSize: 40)
struct ContentView: View {
    @State private var text = ""
    var body: some View {
        VStack {
            HighlightedTextEditor(text: $text, highlightRules: [])
            .introspect { editor in
                editor.textView.font = baseFont
            }
        }
        .padding()
    }
}
```
